### PR TITLE
Honda Pilot 2017 Port

### PR DESCRIPTION
**Pull request by vanillagorillaa**
Updated to reflect var changes in 0.3.9. Still needs to be tested.

### CODE REVIEW

**Comment by energee** [comment]
Pilot is _kind of_ working, steering PID values seem to be off. https://comma.slack.com/files/U81JA444X/F8FVA32JU/img_8764.mov

**Comment by vanillagorillaa** [comment]
@rbiasini This should be good for alpha support

**Comment by mcampa** [comment]
I've been driving with this branch for a few weeks now, and this are a few notes:

- If I switch from D to S (sport) gear OP does not work and displays a "Gear not D" message. It should work on S gear.

- I get a permanent message at the top center of the screen `STEER FAULT RESTART THE CAR` since the 0.4 update. However, OP works fine. The alert key is `steerUnavailablePermanent` but I couldn't find where is triggered in the code.

- Still getting a little bit of steering wheel oscillation when coming out of turns. This happens only ~10% of the turns.

**Comment by energee** [comment]
* OP does not support "Sport" mode, this is George's original code from ILX/Civic. Since it is opensource, you could change this but support for only drive mode is part of their safety requirements.

* @vanillagorillaa Can we check if a variable changed? Maybe this is due to an "error" variable changing to "event"

* This sounds about on-par with supported cars. This will get better as the controls improve.

**Comment by vanillagorillaa** [comment]
@energee  Updated port to reflect @addibble 's Identification of the steer fault value. 
@mcampa We will edit the control values here shortly, what values are you currently using?

**Comment by mcampa** [comment]
This branch is broken right now. After pulling the latest changes I get errors in the car display. I'm trying to find the issue right now.

**Comment by vanillagorillaa** [comment]
@mcampa My branch does not have the new refactored DBC file for the Pilot yet

File: selfdrive/car/honda/carstate.py [hunk 1]
```diff
@@ -131,6 +131,8 @@ def get_can_signals(CP):
                 ("EPB_STATE", "EPB_STATUS", 0),
                 ("BRAKE_HOLD_ACTIVE", "VSA_STATUS", 0)]
     checks += [("EPB_STATUS", 50)]
+  elif CP.carFingerprint == CAR.ODYSSEY:
```
Comment (mcampa) [inline selfdrive/car/honda/carstate.py]: do you mean `CAR.PILOT`?

Comment (vanillagorillaa) [reply selfdrive/car/honda/carstate.py]: Fixed

**Pull request merged**